riscv-sbi.adoc

```diff
@@ -1674,6 +1677,388 @@ The possible error codes returned in `sbiret.error` are shown in the
+│    └───────────┘     │                       │
+└──────────────────────┘                       │
+            △                                  │
+            │                                  │
+            └─────────Secret Injection─────────┘
+
+....
+
+These functions provide the capabilities and features required to implement
+attestation for usages such as confidential computing, measured boot and
+verified boot.
+
+[#get-capabilities-function,reftext="Get Capabilities"]
+=== Function: Get Attestation and Measurement Capabilities (FID #0)
+
+[source, C]
+----
+struct sbiret sbi_attestation_get_capabilities(unsigned long *caps_addr
+                                               unsigned long caps_size);
+----
+
+Get the SBI implementation attestation and measurement capabilities.
+
+The reported capabilities allow attestation and measurement implementations to
+build platform agnostic frameworks.
+
+The SBI implementation writes an attestation and measurement capabilities
+structure at the `caps_addr` physical address, where the caller must allocate
+`caps_size` bytes.
+The capabilities structure format follows the `ATST_CAPS` format described in
+<<table_attestation_and_measurement_capabilities_format>>.
+
+`caps_size` must be large enough to accomodate for the returned measurement
+capabilities structure. The largest possible capabilities structure is 136
+bytes.
+The actual length for the returned capabilities structure is returned through
+`sbiret.value`.
+
+[#table_attestation_and_measurement_capabilities_format]
+.Attestation and Measurement Capabilities Format Definition (ATST_CAPS)
+[cols="5,2,2,2,5", width=90%, align="center", options="header"]
+|===
+| Field Name            | Offset         | Type           | Size                   | Description
+| TCB_SVN               | 0              | Integer        | 4                      | TCB Security Version Number
+| HASH_ALGORITHM        | 4              | Bitmap         | 1                     a| Measurement hash algorithm:
+
+                                                                                     * 0: SHA-384
+                                                                                     * 1: SHA-512
+| EVIDENCE_FORMATS      | 5              | Bitmap         | 1                     a| Supported evidence X.509 extension identifier bitmap:
+
+                                                                                     * Bit 0: https://trustedcomputinggroup.org/wp-content/uploads/TCG_DICE_Attestation_Architecture_r22_02dec2020.pdf[`DiceTcbInfo`]
+                                                                                     * Bit 1: https://trustedcomputinggroup.org/wp-content/uploads/TCG_DICE_Attestation_Architecture_r22_02dec2020.pdf[`DiceMultiTcbInfo`]
+                                                                                     * Bit 2: https://pigweed.googlesource.com/open-dice/[`OpenDice`]
```

Comment:
As this is a bitmap, you should point out that all other bits are reserved for now.

Reply:
This needs to be marked as resolved. It does appear that the bitmap was added below.